cpp/ql/test/library-tests/ir/range-analysis/test.cpp

```diff
@@ -49,3 +49,13 @@
     return 0;
   }
 
+  void* f3_get(int n);
```

Comment:
Just because I'm curious: Is the FP still there if the return type is changed to `int`?

Reply:
Yes.